data/transactions/logic/tlhc.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+
+import argparse
+import base64
+import hashlib
+import os
+import sys
+
+template = '''txn CloseRemainderTo
```

Comment:
Should there be a cap on the fee here? (Otherwise, as a counterparty, I can destructively tamper with the contract by setting the fee to be very high, which would drain the sender of funds.)